backend/app/services/summary_service.py

```
@@ -0,0 +1,627 @@
+                document.summary = summary_data
+                flag_modified(document, "summary")
+                self.db.commit()
+            except Exception as commit_error:
+                logger.warning(
+                    f"[SummaryService] Failed to save error status: {commit_error}"
+                )
+                self.db.rollback()
+            return None
+
+    async def get_document_summary(
+        self, document_id: int, user_id: int
```

Ruff (0.14.10)

Unused method argument: `user_id`

(ARG002)